packages/runtime-core/src/components/Teleport.ts

```diff
@@ -282,6 +285,19 @@ export const TeleportImpl = {
   hydrate: hydrateTeleport,
 }
 
+function hasNotDisabledTeleport(vnodes: VNode[]): Boolean {
```

Comment:
I think this is the source of the `lint-and-test-dts` failure:
```suggestion
function hasNotDisabledTeleport(vnodes: VNode[]): boolean {
```